scripts/check_python_module

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+#
+# Check if a compatible version of a python module is installed
+#
+# usage: check_python_module <module> <min_version>
+#
+
+from __future__ import print_function
+import sys
+import importlib
+import argparse
+
+parser = argparse.ArgumentParser(description="Check presence and version of a Python module")
+parser.add_argument("module", help="python module to check")
+parser.add_argument("min_version", help="minimum version")
+parser.add_argument("-v", "--verbose", action="store_true")
+args = parser.parse_args()
+
+modname = args.module
+version = args.min_version
+
+#
+# See if numpy is installed in the first place
```

Comment:
Yeah.  That is obviously wrong.  I changed it to say "see if module is installed in the first place"